flight_planning/v1/geotemporal.yaml

```diff
@@ -65,11 +65,12 @@ components:
         reference:
           description: >-
             A code indicating the reference for a vertical distance. See AIXM
-            5.1 and FIXM 4.2.0.
+            5.1 and FIXM 4.2.0 for W84 and SFC.
           type: string
           enum:
             - W84
             - SFC
+            - Takeoff
```

Comment:
Is this change intended?

Reply:
Yes -- this is the most realistic datum for, e.g., consumer DJI aircraft to use in reports.